### DISCUSSION TITLE
[Esame di prova 30 dicembre 2023] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#389] [Esami - Viaggi/Piazza]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fec030e8-4a19-4703-a11e-d9d47169e375)

**Comment by Elia-Belli** [comment]
**(1)** Ci conviene svolgere i prodotti in caso di cicli non disgiunti:

$$\alpha=(4\ 6)(1\ 7\ 3)(1\ 2\ 5)=\begin{pmatrix}1&2&3&4&5&6&7&8\\
2&5&1&6&7&4&3&8\end{pmatrix}=(1\ 2\ 5\ 7\ 3)(4\ 6)$$

$$\beta = (8\ 7\ 5\ 4\ 3)(1\ 2)$$

$$\gamma =(1\ 2\ 3)(8\ 6\ 4)(8\ 7)=\begin{pmatrix}1&2&3&4&5&6&7&8\\
2&3&1&8&5&4&6&7\end{pmatrix}=(1\ 2\ 3)(4\ 8\ 7\ 6)$$

**(2)** Per determinare il segno (parità) di ognuna decomponiamo in trasposizioni i cicli: 
$$\alpha=(1\ 2\ 5\ 7\ 3)(4\ 6)=(1\ 3)(1\ 7)(1\ 5)(1\ 2)(4\ 6)\Rightarrow \alpha\ è\ dispari$$

$$\beta=(8\ 7\ 5\ 4\ 3)(1\ 2)=(8\ 3)(8\ 4)(8\ 5)(8\ 7)(1\ 2)\Rightarrow \beta\ è\ dispari $$

$$\gamma=(1\ 2\ 3)(4\ 8\ 7\ 6)=(1\ 3)(1\ 2)(4\ 6)(4\ 7)(4\ 8)\Rightarrow \gamma\ è\ dispari$$

Alternativamente, ricordando che cicli di lunghezza dispari sono pari e viceversa, e che valgono le regole dell'aritmetica sugli interi per il segno:
$\alpha,\ \beta,\ \gamma$ sono dispari perchè prodotto di un ciclo dispari per un ciclo pari.

**(3)** Due permutazioni sono coniugate se hanno stessa struttura ciclica, quindi $\alpha$ e $\beta$ sono coniugate mentre $\gamma$ no.
La permutazione $\tau$ che le coniuga, ovvero tale che $\alpha=\tau \beta \tau^{-1}$, deve obbligatoriamente mappare in tal modo i seguenti elementi:

$$\tau(8)=1\ \ \tau(7)=2\ \ \tau(5)=5\ \ \tau(4)=7\ \ \tau(3)=3\ \ \tau(1)=4\ \ \tau(2)=6\ \ $$

Inoltre $\tau$ è unica in questo caso siccome $\alpha,\beta$ hanno un unico punto fisso, quindi in forma matrice:

$$\tau=\begin{pmatrix}1&2&3&4&5&6&7&8\\
4&6&3&7&5&8&2&1\end{pmatrix}$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/cfe45287-7855-4505-8936-62c877460c69)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-06 220933](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/0f55248f-cfad-46a7-9f7f-02a0734e9ba1)
![Immagine 2024-01-06 220952](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/0257b21a-74e2-4faa-bd04-8bbe3e6dc78e)
![Immagine 2024-01-06 221002](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/c2e7b4b0-3d11-4931-b10b-53ff4e4e708a)
![Immagine 2024-01-06 221012](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/b22cf07f-b89d-4b9a-9a14-9740eff3b576)
